src/jwt.rs

```diff
@@ -50,6 +50,7 @@ fn token_parsing_error() -> MomentoError {
 }
 
 #[cfg(test)]
+#[allow(clippy::unwrap_used)]
```

Comment:
Same here.

Reply:
Same thing, addressed in https://github.com/momentohq/client-sdk-rust/commit/5a8b158f5f3cc26c7a04aedb4f5c9cb6d51e387e